src/main/java/frc/robot/Constants.java

```diff
@@ -162,40 +161,46 @@ public static class OperatorConstants {
   }
 
   public static class TrapElvConstants {
-    // IDs | TODO: get real device IDs
-    public static final int WRIST_MOTOR_ID = 20;
-    public static final int ROLLER_MOTOR_ID = 21;
+    // Wrist
+    public static final int WRIST_MOTOR_ID = 12;
+    public static final int WRIST_ENCODER_ID = 25;
+    public static final int SOURCE_BREAK_ID = 1;
+    public static final int GROUND_BREAK_ID = 2;
+
+    public static final double[] WRIST_PID = {0.03, 0, 0, 0};
```

Comment:
Per what I said in person, it is generally better to create a small class for this sort of data, as it makes it clearer to the reader what each value means.

That said, this would be a non-trivial change, so we won't worry about it for right now.